### ISSUE TITLE
 Submit file to cuckoo with arguments

### ISSUE THREAD

**Issue by RazviAlex** [cuckoosandbox/cuckoo#2822]
When execute a file, needs a parameter to run it, for example just an 'aaa' or anything. 

I have looked at similar doubts and found nothing. What I found in the guide is to put this:
`cuckoo submit --options arguments=--aaa /path/to/binary.exe`
doesn't work, cuckoo just execute the file and not write 'aaa'

Also try with:
`cuckoo submit /path/to/binary.exe --options arguments "aaa"` 
and the same

How would I go about submit the file to cuckoo so that it runs?
Thank you!

**Comment by ryanbekabe** [comment]
Use API here: https://cuckoo.readthedocs.io/en/latest/usage/api/#tasks-create-file

**Comment by RazviAlex** [comment]
Thank you @ryanbekabe I appreciate it. 
but there is no other way to do it? don't know how to do it in the normal way, so with API REST I totally confuse how to submit with arguments (the only example there just show normal submit) :(

**Comment by ryanbekabe** [comment]
You are right @RazviAlex , the argument function also does not function properly in my place.
https://github.com/ryanbekabe/picstuff/blob/master/cuckoosandboxarguments.png

In my opinion, you have to re-customize the file using SFX (.exe) Zip / RAR for example, by inserting the command you want.

**Comment by RazviAlex** [comment]
You try this sentence @ryanbekabe :  `cuckoo submit /path/to/binary.exe --options arguments="aaa"` with this in my case it show the argument in the log, but just there. The file does not recive any argument

**Comment by ryanbekabe** [comment]
I try your command @RazviAlex, and some other commands:
cuckoo submit pafish.exe --options arguments="aaa"
cuckoo submit --package exe --options arguments=--dosomething pafish.exe
cuckoo submit --package exe --options arguments=--bkb pafish.exe
cuckoo submit --package exe pafish.exe --options arguments=--bkb

I'm also curious about hard-to-implement features like IRMA and Remote Control guacamole directly when files are being analyzed. :)
